### DISCUSSION TITLE
Pre-populate the docker managed volume with SCAP/CERT/NVT/GVM data.

### DISCUSSION THREAD

**Discussion by austinsonger** [Secure-Compliance-Solutions-LLC/GVM-Docker#162] [Feature Request]
This should be faster than syncing everything from scratch.

**Comment by obker** [comment]
That would be awesome ! Do you have any idea on how to do that ?

**Reply by austinsonger** [reply]
It is already part of the master branch, this is still in testing.

See the commit from last night https://github.com/Secure-Compliance-Solutions-LLC/GVM-Docker/commit/ed7588e34bfa1a6c85bb88034719599eb79fb640
